cognite/client/_api/agents/agents.py

```diff
@@ -251,6 +251,12 @@ def chat(
         Given a user query, the Atlas AI agent responds by reasoning and using the tools associated with it.
         Users can ensure conversation continuity by including the cursor from the previous response in subsequent requests.
 
+        Note:
+            Agents can take time to execute and may not finish within the default timeout.
+            If you experience timeout errors, you can increase the timeout by setting:
+
+                >>> client.config.timeout = 60  # Set timeout to 60 seconds
+
```

Comment:
@ks93 If you change `self._post` to `self._do_request("POST", ...)`, I believe you can pass an override `timeout`. Then I would suggest agents API calls were appropriate get bumped to 60 sec and even accept timeout directly, `timeout: int = 60`